docs/data/joy/components/typography/typography.md

```diff
@@ -80,26 +91,9 @@ const theme = extendTheme({
 });
 ```
 
-### Levels
-
-The `Typography` component has access to the typographic level scale defined in the theme.
-Use the `level` prop to control the scale value.
-
-:::warning
-Keep in mind that each level renders a specific HTML tag (e.g. "h1" renders as an `<h1>` element, "body1" renders as a `<p>`, etc.)
-:::
-
-{{"demo": "TypographyScales.js"}}
-
-### Decorators
-
-Use the `startDecorator` and/or `endDecorator` props to add supporting icons or elements to the typography.
-
-{{"demo": "TypographyDecorators.js"}}
-
 ### Nested typography
 
-Nested `Typography` components will render as a `<span>` tag by default, unless a value for the `component` prop in each component is specified.
+Nested Typography components will be rendered as a <span> element by default, unless a custom semantic element is specified using the component prop in each component.
```

Comment:
As a general rule, anything wrapped in <>s should get back ticks. Same goes for names of props.

```suggestion
Nested Typography components will be rendered as a `<span>` element by default, unless a custom semantic element is specified using the `component` prop in each component.
```